packages/core/project.json

```diff
@@ -63,7 +63,8 @@
                 "commands": [
                     "tsc --outDir ../../temp --declaration --emitDeclarationOnly",
                     "api-extractor run --config api-extractor/checkout-sdk.json & api-extractor run --config api-extractor/checkout-button.json & api-extractor run --config api-extractor/embedded-checkout.json & api-extractor run --config api-extractor/internal-mappers.json",
-                    "rm -rf ../../temp"
+                    "rm -rf ../../temp",
+                    "nx run hosted-form-v2:build-dts"
```

Comment:
Just out of curiosity, shall we add this command here as we won't need to generate these files for all build since we won't be changing hosted-form-v2 as regularly. Maybe we can keep the type generation as a manual process for now?

Reply:
@animesh1987 Good call out. I think eventually we want to move the command out of `core`. But right now, the compilation of the script is still coupled with `core` and its webpack config. In the other words, the build process for `hosted-form-v2` is not entirely independent from `core` and further work is needed to fully separate them. That's why I think it's better to keep the DTS generation together for now. In terms of the build speed concern, NX should have caching in place to prevent unnecessary rebuild if the source files are unchanged.

Reply:
I think for the builds like this NX loses the caching, something we can verify 👍